aerogear/mobile-cli
# integration golden file fix

**Describe what this PR does and why we need it**:
`get services -o=json` output has partial dynamic data that will vary between clusters. 
This PR adds a change that will strip that data both from expected and actual strings (output, golden file read), it does not modify golden file itself.

## Verification steps
1. [set up local cluster](https://github.com/aerogear/mobile-core/blob/master/docs/walkthroughs/local-setup.adoc)
2. in mobile-cli run `make integration` -it should not FAIL
3. in mobile-core run `make clean`
4. repeat steps 1 and 2
